README.md

```diff
@@ -38,6 +38,20 @@ Now you can interact with Penumbra using `pcli`:
 cargo run --bin pcli -- --help
 ```
 
+Generate keys using:
+
+```
+$ cargo run --bin pcli -- --wallet-location penumbra_wallet.dat generate
+Wallet generated, stored in penumbra_wallet.dat. WARNING: This file contains your private keys. BACK UP THIS FILE!
+Your first address is penumbra_tn001_10ftlcft6c8n95cc5lpwdupt2d86n0td2t55xjwkgnte9jgzlqfp7xlqfnssljygq6fxspvnj5xuc5j0qtd6j398elyhrqugs07r7s8v5n0dpkgweytjqm2gv6hmdef
+```
+
+Keys will be stored in the location you provide via `--wallet-location <path>` or if you specify no location, in `pcli`'s data directory:
+
+* Linux: `/home/alice/.config/pcli/penumbra_wallet.dat`
+* macOS: `/Users/Alice/Library/Application Support/zone.penumbra.pcli/penumbra_wallet.dat`
+* Windows: `C:\Users\Alice\AppData\Roaming\penumbra\pcli\penumbra_wallet.dat`
```

Comment:
Looking at this I worry that someone thinks that they'll have to specify `--wallet-location`, set the wallet in a nonstandard location, and then have to specify it every time.  I'm wondering if we should either (a) not include it in the instructions, falling through to the default, or (b) not support multiple wallet locations at all.

Reply:
Hmm true, let me remove it from the instructions (I feel like the option is useful itself but if we later decide it isn't we can remove it too), will remove this in a commit